uint8 type
float32 pan_min
float32 pan_max
float32 pan_vel
uint8 nsteps
